Extensions/TileMap/tilemapruntimeobject.js

```diff
@@ -0,0 +1,242 @@
+/**
+ * Displays a tilemap (mapeditor.org supported).
+ * @memberof gdjs
+ * @class TileMapRuntimeObject
+ * @extends RuntimeObject
+ */
+gdjs.TileMapRuntimeObject = function(runtimeScene, objectData) {
+  gdjs.RuntimeObject.call(this, runtimeScene, objectData);
+  this._frameElapsedTime = 0;
+
+  /** @type {number} */
+  this._opacity = objectData.content.opacity;
+  /** @type {string} */
+  this._tilemapJsonFile = objectData.content.tilemapJsonFile;
+  /** @type {string} */
+  this._tilemapAtlasImage = objectData.content.tilemapAtlasImage;
+  /** @type {string} */
+  this._displayMode = objectData.content.displayMode;
+  /** @type {number} */
+  this._layerIndex = objectData.content.layerIndex;
+  /** @type {number} */
+  this._animationSpeed = objectData.content.animationSpeed;
+  /** @type {number} */
+  this._animationFps = objectData.content.animationFps;
+
+  if (this._renderer)
+    gdjs.TileMapRuntimeObjectRenderer.call(this._renderer, this, runtimeScene);
+  else
+    this._renderer = new gdjs.TileMapRuntimeObjectRenderer(this, runtimeScene);
+
+  // *ALWAYS* call `this.onCreated()` at the very end of your object constructor.
+  this.onCreated();
+};
+
+gdjs.TileMapRuntimeObject.prototype = Object.create(
+  gdjs.RuntimeObject.prototype
+);
+gdjs.registerObject('TileMap::TileMap', gdjs.TileMapRuntimeObject);
+
+gdjs.TileMapRuntimeObject.prototype.getRendererObject = function() {
+  return this._renderer.getRendererObject();
+};
+
+gdjs.TileMapRuntimeObject.prototype.update = function(runtimeScene) {
+  if (this._animationSpeed <= 0) return;
```

Comment:
You must also protect against this._animationFps being 0, because this would generate a division by 0.